frontend/src/old-pages/Configure/Queues/__tests__/SubnetMultiSelect.test.tsx

```diff
@@ -0,0 +1,180 @@
+import {render} from '@testing-library/react'
+import {Subnet} from '../queues.types'
+import {SubnetMultiSelect} from '../SubnetMultiSelect'
+import wrapper from '@cloudscape-design/components/test-utils/dom'
+
+const mockSelectedSubnets: string[] = ['subnet-1']
+const mockSubnets: Subnet[] = [
+  {
+    SubnetId: 'subnet-1',
+    AvailabilityZone: 'AZ-a',
+    AvailabilityZoneId: 'AZ-a-id',
+    VpcId: 'VPC-1',
+  },
+  {
+    SubnetId: 'subnet-2',
+    AvailabilityZone: 'AZ-b',
+    AvailabilityZoneId: 'AZ-b-id',
+    VpcId: 'VPC-1',
+  },
+  {
+    SubnetId: 'subnet-3',
+    AvailabilityZone: 'AZ-c',
+    AvailabilityZoneId: 'AZ-c-id',
+    VpcId: 'VPC-2',
+  },
+]
+
+const mockUseState = jest.fn()
+
+jest.mock('../../../../store', () => ({
```

Comment:
suggestion: this mocking part can be really simplified with no monkey patching at all! Take for example what has been done in `storage.test.tsx`
```typescript
const mockStore = mock<Store>()
 mockStore.getState.mockReturnValue({
          app: {
            wizard: {
              vpc: 'VPC_SELECTED',
          },
        })

 const {container} = render(
     <Provider store={mockStore}>
        <SubnetMultiSelect
          value={mockSelectedSubnets}
          onChange={mockedOnChange}
        />
     </Provider>
)
```